libr/anal/p/anal_sbpf.c

```diff
@@ -500,6 +534,18 @@ static bool sbpf_analyze_strings(RAnal *anal) {
 				}
 			}
 
+			// Add comment at the instruction address showing the string value
+			if (ref->xref_addr != UT64_MAX && strlen(comment_str) > 0) {
```

Comment:
no need to strlen > 0 you can just check if the first byte is != 0

```suggestion
			if (ref->xref_addr != UT64_MAX && *comment_str) {
```